apps/demo/src/DemoApp.tsx

```diff
@@ -7,17 +7,11 @@ import MockApp from './MockApp';
 function DemoApp() {
   return (
     <Router>
-      <Switch>
-        <Route path="/mock">
-          <MockApp />
-        </Route>
-        <Route path="/hsds">
-          <HsdsApp />
-        </Route>
-        <Route path="/">
-          <H5GroveApp />
-        </Route>
-      </Switch>
+      <Routes>
+        <Route path="/" element={<H5GroveApp />} />
+        <Route path="mock" element={<MockApp />} />
+        <Route path="hsds" element={<HsdsApp />} />
+      </Routes>
```

Comment:
`react-router-dom` v6 is a complete rewrite, which includes very nice enhancements, such as relative paths in `Route` in `Link`, no need for `exact` or for a specific ordering of the routes (`Switch` is replaced with `Routes`), routes can all be declared in the same component and nested (parent routes can include an `Outlet` element to say where to render their nested routes).

- https://remix.run/blog/react-router-v6
- https://reactrouter.com/docs/en/v6/upgrading/v5
- https://reactrouterdotcom.fly.dev/docs/en/v6/getting-started/overview#nested-routes

Reply:
Nice.

With the previous version, if a path was not recognized it would fall back to `/` (i.e. `localhost/foo` would fallback to `localhost/`). Does the new version change this behaviour ?

Reply:
It was displaying a blank page. I added a `*` route with `<Navigate to="/" />`.

Reply:
Great, at least the behaviour is explicit now